newlib/libc/machine/riscv/strcmp.S

```diff
@@ -46,12 +46,14 @@ strcmp:
   .macro check_one_word i n
     REG_L a2, \i*SZREG(a0)
     REG_L a3, \i*SZREG(a1)
-
+#ifdef __riscv_zbb
+    orc.b t2, t0
```

Comment:
Reading the adjacent code, it seems like this should be
```
    orc.b t0, a2
```
that is, t0 should contain 0xff in each non-zero byte of a2.

Reply:
Edited on the same commit